src/content/development/contribution-guide/_index.en.md

```diff
@@ -78,15 +78,27 @@ To be expanded
 
 ### Unplanned Work
 
-To be expanded
+During the work on a release, there will be tasks which the team have to add to the projects which was unknown at the planning phase.
+There are 3 type of those tasks:
+
+* [Bugs](#bugs).
+* Ongoin maintenance.
+* New [Epics](#epics) or small independent tasks that need to start work on.
+
+Bugs do not need to be part of a release planning, and the two other types are listed below:
 
 #### Ongoing Maintenance
```

Comment:
I wonder if it would make more sense to put this under the "tasks, bugs, epics" since it's not strictly related to planning or not, its like bugs that are continuous.

Reply:
Which part do you consider moving to "tasks, bugs, epics"?
The first ("tasks, bugs, epics") is a definition, here is in relation to the  work process

Reply:
@mkolesnik - what do suggest to do here?